packages/backend/src/apps/m365-excel/actions/create-table-row/index.ts

```diff
@@ -135,37 +136,25 @@ const action: IRawAction = {
       await RATE_LIMIT_FOR_RELEASE_ONLY_REMOVE_AFTER_JULY_2024($.user?.email, $)
     }
 
-    const { fileId, tableId } = $.step.parameters
+    const parametersParseResult = parametersSchema.safeParse($.step.parameters)
 
-    // Validation to prevent path traversals
-    validateDynamicFieldsAndThrowError(String(fileId), String(tableId), $)
-
-    const columnValues = ($.step.parameters.columnValues as IJSONObject[]) ?? []
-    if (columnValues.length === 0) {
-      $.setActionItem({
-        raw: {
-          success: true,
-        },
-      })
-      return
+    if (parametersParseResult.success === false) {
```

Comment:
nitpick
```suggestion
    if (!parametersParseResult.success) {
```

Reply:
because the return type is a discriminated union on success, we cant use that syntax here